<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnotherWindow</class>
 <widget class="QWidget" name="AnotherWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>813</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="backNaturalNum">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>791</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Натуральные числа</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Первое число:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Второе число:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditFirstNatural">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>321</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>20</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Цифра:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditNumeral">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>20</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditSecondNatural">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>50</y>
      <width>321</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="push_SUB_NDN_N">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>228</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>SUB_NDN_N</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_NZER_N_B">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>190</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>NZER_N_B</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_ADD_1N_N">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>188</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ADD_1N_N</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_DIV_NN_N">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>228</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>DIV_NN_N</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_MOD_NN_N">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>228</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>MOD_NN_N</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_MUL_ND_N">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>188</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>MUL_ND_N</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_MUL_Nk_N">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>188</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>MUL_Nk_N</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_MUL_NN_N">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>228</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>MUL_NN_N</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_DIV_NN_Dk">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>228</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>DIV_NN_Dk</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_ADD_NN_N">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>188</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ADD_NN_N</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_SUB_NN_N">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>188</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>SUB_NN_N</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_GCF_NN_N">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>228</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>GCF_NN_N</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_LCM_NN_N">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>228</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>LCM_NN_N</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_COM_NN_D">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>190</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>COM_NN_D</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>330</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Ответ:</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="answerNatural">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>320</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="faqN">
   <property name="geometry">
    <rect>
     <x>774</x>
     <y>10</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>?</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
